packages/language/languages/en.ts

```diff
@@ -197,7 +197,7 @@ export default {
       transferAll: `TRANSFER EVERYTHING`,
       balance: (balance) => `Your current ETH balance is ${typeof balance === 'string' ? balance.slice(0,8) :''}`,
       ethAddress: `Ethereum Address`,
-      txCost: (cost, currency) => `The current transaction cost is ${currencySymbols(currency)}${cost}`,
+      txCost: (ethCost, currencyCost) => `Transaction cost is ${ethCost} ETH (${currencyCost})`,
```

Comment:
we should keep the `currencySymbols(currency)` code instead of hard-coding the currency

Reply:
The symbol is not "hard-coded", it is already baked into the `currencyCost` string.

Are you referring to "ETH" ?

Reply:
nope, I misread it, you're right